src/components/TaskPreview.js

```diff
@@ -50,10 +67,114 @@ function EventReview(props) {
         <Typography variant="body2" color="textSecondary" component="p">
           Hello the world. Just do something.... What... Exercise is good for your health
```

Comment:
Can we fix this block to not have this place holder text?

Reply:
yas

Reply:
There needs to be a bit of placeholder text, "Expand for details" or something? maybe?

Reply:
I take that back, its fine without for now.